### DISCUSSION TITLE
Instructor Class Assignment and Distribution Preferences

### DISCUSSION THREAD

**Discussion by aytan97** [UniTime/unitime#176] [General]
Hi,

I hope you're doing well.

I would like to ask for your advice regarding distribution preferences inside Instructional Offerings.

Inside Instructional Offerings we have created the course, and added configuration based on the enrollment count with 2x75(twice a week).
Some instructors have more than 4 lectures. We want them to teach at least two classes on the same day and not more than 3 weekdays usage (for 4 lecture cases) to get assigned classes. Overall we want schedules like MW(1st class), MW(2nd class), TTH(3rd class), TTH(4th class),  and in the same-day classes the time gap should not be longer than 15 minutes.

We have added distribution preferences like Back-To-Back and Groups of Two. Unfortunately, it gives all classes on the same day like MW x 4

Could you please advise which kind of combination in distribution preferences we can get the desired solution?

Thank you & Regards,
Aytan

**Comment by tomas-muller** [comment]
The Back-To-Back preference using the Groups of Two structure will ensure that the first class is back to back to the second class, the third class is back to back to the fourth. To ensure that they are on different days, you would need to add prohibited Same Days between the 1st and the 3rd class. Alternatively, you can require MW times on the first class and require TTh days on the third class instead.
